### ISSUE TITLE
Make the pre-processing consistent with yolov5

### ISSUE THREAD

**Issue by zhiqwang** [zhiqwang/yolov5-rt-stack#274]
### 🚀 The feature and motivation
As mentioned in https://github.com/zhiqwang/yolov5-rt-stack/issues/265#issuecomment-1008351793, the `YOLOv5.load_from_yolov5()` strategy will introduce minor differences with ultralytics, now there are two ways to solve this problem:

1. Make the [pre-processing](https://github.com/zhiqwang/yolov5-rt-stack/blob/949fa0b08dd336c21dc051594941af457a312b21/yolort/models/transform.py#L38) behave the same with yolov5. The advantage of this approach is that there is no need to write code on the C++ backend.
2. Just use `YOLO.load_from_yolov5()` and implement the pre-precessing in the C++ backend. This requires to modify the example we provide. And the main difference between this strategy and the official yolov5 is that we don't need to write the post-processing (`nms`) module on the C++ backend.

In fact, the main ops needed for preprocessing is `interpolate`, yolov5 uses the `cv2.resize` operator, we use `torch.nn.functional.interpolate` in yolort. (And the ops in cv2 can't be traced or scripted.) The implementation of the `interpolate` operator is not entirely consistent across frameworks. We will optimize the implementation of our pre-processing this week and hope that the results will be available next week.

https://github.com/zhiqwang/yolov5-rt-stack/blob/949fa0b08dd336c21dc051594941af457a312b21/yolort/v5/utils/augmentations.py#L126

https://github.com/zhiqwang/yolov5-rt-stack/blob/949fa0b08dd336c21dc051594941af457a312b21/yolort/models/transform.py#L286-L293

### Additional context
https://pytorch.org/blog/introducing-torchvision-new-multi-weight-support-api/#associated-meta-data---preprocessing-transforms

_Originally posted by @zhiqwang in https://github.com/zhiqwang/yolov5-rt-stack/issues/271#issuecomment-1009600778 cc @mattpopovich ._

**Comment by zhiqwang** [comment]
Hi @mattpopovich ,

There is a good news about it, I think we've finally resolved the issue here, and I'll update the comparison documentation in a follow-up PR.

I haven't done a complete test on the speed, but since we are using basically the same operators both before and after the change, it' s likely that the code will not be slower after the change. We will pay more attention to the performance part in subsequent updates. We would welcome additional testing on different devices..

As such I'm closing this thread, and we'd be happy to hear more from you about `yolort`, and feel free too create another ticks if you have more questions.

**Comment by zhiqwang** [comment]
FYI, we've added the doc for comparison between yolov5 & yolort:
https://zhiqwang.com/yolov5-rt-stack/notebooks/comparison-between-yolort-vs-yolov5.html
